examples/features/BSplineInterpolation.ipynb

```diff
@@ -97,7 +97,7 @@
   {
    "cell_type": "markdown",
    "metadata": {},
-   "source": [
+   "collection": [
```

Comment:
this part should not be changed, as it changes metadata of the notebook, resulting in invalid notebook

Reply:
fixed